cardano-cli/src/Cardano/CLI/Type/Common.hs

```diff
@@ -489,10 +489,12 @@ data OutputFormatJsonOrText
   | OutputFormatText
   deriving (Eq, Show)
 
-data AllOutputFormats
-  = FormatJson
-  | FormatText
-  | FormatCBOR
+-- | The output format for ledger outputs.  Ledger outputs are those outputs that have a ledger representation
+-- (ie. CBOR).
+data LedgerOutputFormat
```

Comment:
We need to redesign this and apply this to all commands. Do you consider the `TextEnvelope` format to be a ledger format?